packages/cosmwasm-stargate/src/cosmwasmclient.ts

```diff
@@ -56,7 +62,13 @@ export interface PrivateCosmWasmClient {
 
 export class CosmWasmClient {
   private readonly tmClient: TendermintClient;
-  private readonly queryClient: QueryClient & AuthExtension & BankExtension & WasmExtension;
+  private readonly queryClient: QueryClient &
+    AuthExtension &
+    BankExtension &
+    DistributionExtension &
+    IbcExtension &
+    StakingExtension &
+    WasmExtension;
```

Comment:
Since `queryClient` is private and should remain that way, this only adds code but no feature, right?

Reply:
Good point!